### DISCUSSION TITLE
How to add new type with i18n

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#8111] [Q&A]
I try to add

```tid
title: $:/language/Docs/Types/application/vnd.tldraw+json
description: <<lingo Name $:/plugins/linonetwo/tw-whiteboard/language/>>
name: application/vnd.tldraw+json
group: Image
group-sort: 1
```

in my whiteboard plugin, but result is

![图片](https://github.com/Jermolene/TiddlyWiki5/assets/3746270/3036028c-f32f-4d69-961e-fe5ebbac79b4)

And it seems impossible to localize "Image" "Text" , because markdown plugin's type is not grouped with Chinese of "Text".

**Comment by Jermolene** [comment]
Hi @linonetwo at the moment, tiddlers prefixed `$:/language/Docs/Types/` are cloned within each language plugin, and do not use the lingo macro.

**Reply by linonetwo** [reply]
And plugin can't insert tiddler into language plugin (that's why we upgrade the lingo macro). So this is impossible with current design.

But I'm not going to work on refactor that recently, I'm going to play with whiteboard plugin for a while.

**Reply by Jermolene** [reply]
Hi @linonetwo a plugin can still provide a tiddler prefixed `$:/language/Docs/Types/`; there would only be an issue if another plugin were trying to redefine the same entry, and that would be problematic however we approach things.
